### PR TITLE
Front end - remove mnemonic from LocalStorage

### LINKED ISSUE #50: Never store Mnemonic in LocalStorage
We are already storing Mnemonic in LocalStorage for a very brief amount of time. If possible, we should never store it in the LocalStorage

### PR DESCRIPTION
**Pull request by 1devNdogs**
Fixes #50

### CODE REVIEW

File: src/lib/client.ts [hunk 1]
```diff
@@ -103,12 +104,10 @@ export class Client {
   }
 
   // If the app is started for the first time, the client will be started from PlottingProgress page.
-  // In this case, farmerPublicKey and mnemonic must be stored and clear stored farmedBlocks if exist (clearStoredBlocks).
-  // -
   // If the app is started for the second time, the client will be started from Dashboard page.
-  // In this case, farmerPublicKey and mnemonic already exist and just need to load the stored farmedBlocks (loadStoredBlocks).
-  public async init(farmerPublicKey?: string): Promise<void> {
+  public async init(farmerPublicKey?: string, generatedPk?: string): Promise<void> {
```
Comment (nazar-pc) [inline src/lib/client.ts:108]: Mnemonic is not a private key

**Review by nazar-pc — DISMISSED** [review]

File: src/lib/client.ts (hunk 1 above)
Comment (1devNdogs) [reply src/lib/client.ts]: Updated variable and method names

**Comment by ozgunozerk** [comment]
Seems fine to me, although tests are failing

**Review by ozgunozerk — DISMISSED** [review]

**Comment by 1devNdogs** [comment]
> Seems fine to me, although tests are failing

@ozgunozerk. It was a lint problem, Solved it and also fixed a few other warnings.

**Review by ozgunozerk — APPROVED** [review]

**Pull request merged**